src/main/java/io/molnarsandor/trelloclone/util/swagger/SwaggerConfig.java

```diff
@@ -0,0 +1,85 @@
+package io.molnarsandor.trelloclone.util.swagger;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.annotations.ApiIgnore;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.service.ApiKey;
+import springfox.documentation.service.AuthorizationScope;
+import springfox.documentation.service.SecurityReference;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spi.service.contexts.SecurityContext;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger.web.UiConfiguration;
+import springfox.documentation.swagger.web.UiConfigurationBuilder;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.util.Collections;
+import java.util.List;
+
+import static springfox.documentation.builders.PathSelectors.regex;
+
+@EnableSwagger2
+@Configuration
+public class SwaggerConfig {
```

Comment:
Swagger újabb alternatívája az OpenAPI, kicsit flexibilisebb és erősebb a kifejező ereje, ha van időd mindenképpen nézz meg azt is. :)